dockerfiles/docker-compose-webpack.yml

```diff
@@ -0,0 +1,27 @@
+# docker-compose-webpack.yml starts Webpack dev server for hot reload of built
+# static assets.
+version: '3'
+
+services:
+
+  webpack:
+    image: node:12
+    working_dir: /usr/src/app/checkouts/ext-theme
+    environment:
+      - NODE_ENV=development
+    ports:
+      - "10001:10001"
```

Comment:
Do we really need exposing this port int the host?

Reply:
Yes, client requests for the hot reload files go directly to webpack dev server